lib/screens/cow_characteristics_page.dart

```diff
@@ -1,20 +1,18 @@
 import 'package:flutter/material.dart';
-import '../widgets/custom_text_field.dart';
-import '../widgets/custom_dropdown_field.dart';
-import '../models/cow_characteristics_model.dart';
-import '../constants/cow_characteristics_constants.dart';
-import '../../utils/nutrition_calculator.dart';
 import '../models/cow_requirements_model.dart';
 import '../screens/feed_formula_page.dart';
+import '../screens/cow_characteristics_content.dart';
 
 class CowCharacteristicsPage extends StatefulWidget {
-  const CowCharacteristicsPage({super.key});
-
   @override
   _CowCharacteristicsPageState createState() => _CowCharacteristicsPageState();
 }
 
 class _CowCharacteristicsPageState extends State<CowCharacteristicsPage> {
+  bool _showFeedFormula = false;
```

Comment:
From your PR description, and the code I've read, I am not convinced that this is the best way. I think that this will create a lot of technical debt if the project grows.  (ie. if we add a couple of screens to any of the steps in the navigation bar. (All of a sudden, a single bool for the history is insufficient, and we need to invent adhoc datastructures to capture everything.)

If I were implementing, I would take an approach shown by the last demo on this page:
https://api.flutter.dev/flutter/material/NavigationBar-class.html

This basically creates a history for each tab. If it were just for this app, then I might be tempted to leave as-is. However I *think* it makes sense to use the more general solution, because we're looking to build a suite of 3 very similar apps. I am not as informed as I'd like to be to make this call though. So if you think I am making the wrong call, then it's possible there is an issue I am not aware of.